docs/admin-manual/ep-policy-configuration.rst

```diff
@@ -667,90 +667,19 @@ include
-job's environment to track process families on a best-effort basis. This
-usually works well, but it can falter for certain applications or for
-jobs that try to evade detection.
-
-Jobs that run with a user account dedicated for HTCondor's use can be
-reliably tracked, since all HTCondor needs to do is look for all
-processes running using the given account. Administrators must specify
-in HTCondor's configuration what accounts can be considered dedicated
-via the :macro:`DEDICATED_EXECUTE_ACCOUNT_REGEXP` setting. See
-:ref:`admin-manual/security:user accounts in htcondor on Unix platforms` for
-further details.
-
-Ideally, jobs can be reliably tracked regardless of the user account
-they execute under. This can be accomplished with group ID-based
-tracking. This method of tracking requires that a range of dedicated
-group IDs (GID) be set aside for HTCondor's use. The number of GIDs that
-must be set aside for an execute machine is equal to its number of
-execution slots. GID-based tracking is only available on Linux, and it
-requires that HTCondor daemons run as root.
-
-GID-based tracking works by placing a dedicated GID in the supplementary
-group list of a job's initial process. Since modifying the supplementary
-group ID list requires root privilege, the job will not be able to
-create processes that go unnoticed by HTCondor.
-
-Once a suitable GID range has been set aside for process tracking,
-GID-based tracking can be enabled via the
-:macro:`USE_GID_PROCESS_TRACKING` parameter. The minimum and
-maximum GIDs included in the range are specified with the
-:macro:`MIN_TRACKING_GID` and :macro:`MAX_TRACKING_GID` settings. For
-example, the following would enable GID-based tracking for an execute
-machine with 8 slots.
-
-.. code-block:: text
-
-    USE_GID_PROCESS_TRACKING = True
-    MIN_TRACKING_GID = 750
-    MAX_TRACKING_GID = 757
-
-If the defined range is too small, such that there is not a GID
-available when starting a job, then the *condor_starter* will fail as
-it tries to start the job. An error message will be logged stating that
-there are no more tracking GIDs.
-
-GID-based process tracking requires use of the :tool:`condor_procd`. If
-:macro:`USE_GID_PROCESS_TRACKING` is true, the :tool:`condor_procd` will be used
-regardless of the :macro:`USE_PROCD` setting.
-Changes to :macro:`MIN_TRACKING_GID` and :macro:`MAX_TRACKING_GID` require a full
-restart of HTCondor.
-
 .. _resource_limits_with_cgroups:
 
-Cgroup-Based Process Tracking
+Cgroup Based Process Tracking
 '''''''''''''''''''''''''''''
 
 :index:`cgroup based process tracking`
 
-A new feature in Linux version 2.6.24 allows HTCondor to more accurately
-and safely manage jobs composed of sets of processes. This Linux feature
-is called Control Groups, or cgroups for short, and it is available
-starting with RHEL 6, Debian 6, and related distributions. Documentation
+Linux version 2.6.24 introduced kernel control groups, called cgroups
```

Comment:
You date cgroups v1 by the Linux kernel version number and cgroups v2 by the first RHEL to support it.  The latter is probably of more interest to our audience; consider being consistent about this, or just dropping the introduction date entirely, e.g., "All supported Linux platforms support kernel control groups v1; most (starting with Enterprise Linux 8) also support cgroups v2."

Reply:
+1